benchexec/tools/verifuzz.py

```diff
@@ -49,8 +49,18 @@ def determine_result(self, returncode, returnsignal, output, isTimeout):
         lines = " ".join(output)
         if "VERIFUZZ_VERIFICATION_SUCCESSFUL" in lines:
             return result.RESULT_TRUE_PROP
-        elif "VERIFUZZ_VERIFICATION_FAILED" in lines:
+        elif "FALSE(unreach-call)" in lines:
```

Comment:
This change makes the tool-info module non-backwards compatible. How about checking for both strings?

Reply:
Hi Phillip,

I will check this once with our testing. I think it is doable. Once done, I will re-raise the PR.